### PR TITLE
edgedb: Add edgedb version 2.0.2+61917d6

### PR DESCRIPTION
**Pull request by r15ch13**
Add edgedb version 2.0.2+61917d6

### CODE REVIEW

File: bucket/edgedb.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "jsonpath": "$..packages[-1:].version"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://packages.edgedb.com/archive/x86_64-pc-windows-msvc/edgedb-cli-$version.exe#/edgedb.exe"
+            }
+        },
+        "hash": {
+            "url": "https://packages.edgedb.com/archive/.jsonindexes/x86_64-pc-windows-msvc.json",
```
Comment (rashil2000) [inline bucket/edgedb.json:24]: This URL seems x86_64 specific, so shouldn't it go inside the architecture block?

Comment (r15ch13) [reply bucket/edgedb.json:24]: It's sometimes inside and sometimes it is not. :smile: Examples: 
https://github.com/ScoopInstaller/Main/blob/master/bucket/containerd.json
https://github.com/ScoopInstaller/Main/blob/master/bucket/chronograf.json

Comment (rashil2000) [reply bucket/edgedb.json:24]: Ah yes, pre-existing inconsistencies are there 😅

But I try to put architecture-specific URLs inside, and generic URLs outside, as a convention.

**Pull request merged**